nifi-marklogic-processors/src/test/java/org/apache/nifi/marklogic/processor/QueryMarkLogicIT.java

```diff
@@ -116,8 +127,11 @@ public void collectionQueryWithNoIncomingFlowFile() {
      */
     private void verifySimpleCollectionQueryResult(TestRunner runner) {
         MockFlowFile originalFlowFile = runner.getFlowFilesForRelationship(QueryMarkLogic.ORIGINAL).get(0);
+
         final String query = originalFlowFile.getAttribute("marklogic-query");
-        assertTrue(query.contains("<collection-query><uri>" + TEST_COLLECTION + "</uri></collection-query>"),
+        Fragment xmlQuery = new Fragment(query, Namespace.getNamespace("search", "http://marklogic.com/appservices/search"));
+
+        assertEquals(TEST_COLLECTION, xmlQuery.getElementValue("/search:query/search:collection-query/search:uri"),
```

Comment:
Made this test better by using XPath instead of String contains to verify something.